.gitignore

```diff
@@ -1,6 +1,7 @@
 # place this file @ ~/
 
 ## IDE specific
+submissions/krylenger/.DS_Store
```

Comment:
Modifying global config only for your own project is a bad idea.

Please, do not pollute the global scope :)

Reply:
✅ Done